driver/src/main/java/org/neo4j/driver/Config.java

```diff
@@ -149,6 +163,16 @@ public ConfigBuilder withMinIdleTimeBeforeConnectionTest( long milliSecond )
         }
 
         /**
+         * Enable TLS in all connections with server
+         * @param value
+         */
+        public ConfigBuilder withTLSEnabled( boolean value )
```

Comment:
@zhenlineo From what we discussed, I thought there was going to be three levels: full TLS, TLS without trust, or no TLS at all. Did I misunderstand?